lstchain/mc/sensitivity.py

```diff
@@ -402,11 +410,14 @@ def find_best_cuts_sensitivity(simtelfile_gammas, simtelfile_protons,
 
     # Weight events and count number of events per bin:
     for i in range(0, n_bins_energy):  # binning in energy
+        total_rate_proton_ebin = np.sum(rate_weighted_p[(e_reco_p < energy[i + 1]) & (e_reco_p > energy[i])])
```

Comment:
why is this repeated?

Reply:
a mistake in resolving conflicts, thanks